tests/zap_helper_test.py

```diff
@@ -518,6 +518,28 @@ def test_load_session_error(self, session_mock, isfile_mock):
         with self.assertRaises(ZAPError):
             self.zap_helper.load_session(file_path)
 
+    @patch('zapv2.core.xmlreport')
+    @patch('zapcli.zap_helper.ZAPHelper.write_report')
```

Comment:
It's more that I'd consider `write_report` a "private" method in this case and it's an implementation detail effectively rather than a method in the ZAPHelper class I'd expect other code to reuse. So, I'd generally try to mock the external dependency of the standard `open` method in the tests for `xml_report` and `html_report`, and `write_report` will be tested as a result anyway (but we wouldn't need to touch the tests if that method were renamed or whatnot). But I don't think it's a big deal in this case either way. :)